certified-connectors/Rainbird/apiDefinition.swagger.json

```diff
@@ -0,0 +1,871 @@
+                    },
+                    "relationship": {
+                      "properties": {
+                        "type": {
+                          "type": "string"
+                        }
+                      },
+                      "type": "object"
+                    },
+                    "subject": {
+                      "properties": {
+                        "dataType": {
+                          "type": "string"
+                        },
+                        "type": {
+                          "type": "string"
+                        },
+                        "value": {
+                          "type": "string"
+                        }
+                      },
+                      "type": "object"
+                    }
+                  },
+                  "type": "object"
+                },
+                "factID": {
+                  "type": "string"
+                },
+                "rule": {
+                  "properties": {
+                    "bindings": {
+                      "additionalProperties": true,
+                      "properties": {
+                        "conditions": {
+                          "items": {
+                            "properties": {
+                              "certainty": {
+                                "$ref": "#/definitions/Certainty"
+                              },
+                              "factID": {
+                                "$ref": "#/definitions/factID",
+                                "type": "string"
+                              },
+                              "object": {
+                                "$ref": "#/definitions/Object"
+                              },
+                              "objectType": {
+                                "type": "string"
+                              },
+                              "relationship": {
+                                "$ref": "#/definitions/Relationship",
+                                "type": "string"
+                              },
+                              "salience": {
+                                "type": "number"
+                              },
+                              "subject": {
+                                "$ref": "#/definitions/Subject"
+                              }
+                            },
+                            "type": "object"
+                          },
+                          "type": "array"
+                        }
+                      },
+                      "type": "object"
+                    }
+                  },
+                  "type": "object"
+                },
+                "source": {
+                  "type": "string"
+                },
+                "time": {
+                  "type": "number"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "security": [
```

Comment:
@Mat-Rig Can you please let us know the reason to have this security property for this operation?

Reply:
Hi @sriyen-msft, 
We have the option to secure the response of this endpoint (evidence endpoint). This requires passing a security key in the request's header which I cannot achieve dynamically in Power Automate. I removed this security property as it is option. I also mentioned this option in the endpoint description.